src/core_modules/capture-core/components/ListView/FilterSelectors/filterTypes.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import elementTypes from '../../../../../metaData/DataElement/elementTypes';
+import elementTypes from '../../../metaData/DataElement/elementTypes';
```

Comment:
I am thinking that this might be a NIT one. 
```suggestion
import { dataElementTypes } from '../../../metaData';
```